From 51391285d85955c6d33f2a4bfe86602eb5d230fb Mon Sep 17 00:00:00 2001
From: wanghonghao <wanghonghao@bytedance.com>
Date: Thu, 5 Dec 2019 18:59:18 +0800
Subject: [PATCH] iser: remove `__packed` from struct iser_cm_hdr declaration

`__packed` is not defined previously, and was treated as a varible
declaration.

Signed-off-by: wanghonghao <wanghonghao@bytedance.com>
---
 include/iser-private.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/iser-private.h b/include/iser-private.h
index b8b38db7..b943d34a 100644
--- a/include/iser-private.h
+++ b/include/iser-private.h
@@ -158,7 +158,7 @@ struct iser_tx_desc {
 struct iser_cm_hdr {
 	uint8_t      flags;
 	uint8_t      rsvd[3];
-} __packed;
+};
 
 struct iser_pdu {
 	struct iscsi_pdu              iscsi_pdu;
